src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supplyTotal = 0;
+    private int buyTotal = 0;
```

Comment:
These fields are method implementation details. Seems like it won't work properly if you call the getStatistic method several times. Move them to the method level